arch/xtensa/src/esp32/esp32_wifi_adapter.c

```diff
@@ -350,6 +358,7 @@ static int wifi_coex_set_schm_curr_phase_idx(int idx);
 static int wifi_coex_get_schm_curr_phase_idx(void);
 
 extern void coex_bt_high_prio(void);
+extern int coex_wifi_release(uint32_t event);
```

Comment:
`coex_wifi_release` is already declared in `esp_coexist_internal.h`, so no need to declare it again here.